docs/_data/product/configurations.yml

```diff
@@ -93,15 +93,6 @@
     - variable: "@bs-lg"
       default: "0 @su4 @su12 fade(@black-800, 20%)"
       description:
-    - variable: "@bs-i-sm"
```

Comment:
Removing these is also a breaking change, but these classes aren't used anywhere in production.